app/routes/documentation.js

```diff
@@ -0,0 +1,19 @@
+const { views } = require('../constants/admin')
+const { getEnvironmentVariable } = require('../lib/environment-helpers')
+const { admin } = require('../auth/permissions')
+
+module.exports = {
+  method: 'GET',
+  path: '/documentation',
+  options: {
+    auth: { scope: [admin] },
+    handler: async (_, h) => {
+      const allowedEnvironments = ['dev', 'snd', 'local']
```

Comment:
Can we put the allowedEnvironments in a constant file somewhere as you're using it a couple of times in separate route files